src/analyzer/analyzerthread.cpp

```diff
@@ -126,16 +126,16 @@ void AnalyzerThread::doRun() {
     openParams.setChannelCount(mixxx::kAnalysisChannels);
 
     while (awaitWorkItemsFetched()) {
-        DEBUG_ASSERT(m_currentTrack);
-        kLogger.debug() << "Analyzing" << m_currentTrack->getFileInfo();
+        DEBUG_ASSERT(m_currentTrack.has_value());
```

Comment:
```suggestion
        VERIFY_OR_DEBUG_ASSERT(m_currentTrack.has_value()) {
            continue; 
        }
```
because the has_value check is out of sight, and we don't want to crash in a release version.

Reply:
As above, I don't think any logic changed here, since `has_value()` doesn't have any side effects, I don't think that should be necessary (assuming of course, there wasn't some crash hiding in here before).